### ISSUE TITLE
Suggestion 

### ISSUE THREAD

**Issue by YonaProXD** [SociallyIneptWeeb/AICoverGen#139]
I suggest adding either a tab or a section below 'IA cover' for a history of the converted audio files

**Comment by JackismyShephard** [comment]
@YonaProXD This is a good idea. I have been considering adding something similar for my fork of this repo at : https://github.com/JackismyShephard/ultimate-rvc. What exactly did you have in mind? Perhaps something like a dropdown where you can listen to each generated cover song? If you have some other suggestion you are more than welcome to raise an issue in my repository as well :)

**Comment by YonaProXD** [comment]
@JackismyShephard Yes, a dropdown menu where it shows Song Name (Model name), something like what Kits -- AI does.
